### PR TITLE
Add Info button to Project List

### PR DESCRIPTION
**Pull request by michaelphagen**

### CODE REVIEW

**Comment by michaelphagen** [comment]
I am having an issue with the SQL side of things so I can't generate the list until I fix that, but this is the template UI I made before for the project list

File: src/components/ProjectListItem.tsx [hunk 1]
```diff
@@ -1,7 +1,21 @@
...
+import {
+  ListItem,
+  ListItemText,
+  Checkbox,
+  makeStyles,
+  IconButton,
+} from "@material-ui/core";
 import Project from "../calendar/Project";
+import InfoIcon from "@material-ui/icons/Info";
+import Divider from "@material-ui/core/Divider";
```
Comment (dwmorrin) [inline src/components/ProjectListItem.tsx:12]: when these material/core imports come up, put them in the import list that's already happening above

File: src/components/ProjectListItem.tsx [hunk 2]
```diff
@@ -1,7 +1,21 @@
...
+
+const useStyles = makeStyles((theme) => ({
+  space: {
+    margin: theme.spacing(0),
+  },
+}));
```
Comment (dwmorrin) [inline src/components/ProjectListItem.tsx:18]: I did not notice any effect from removing this

File: src/components/ProjectListItem.tsx [hunk 3]
```diff
@@ -12,15 +26,26 @@ const ProjectListItem: FunctionComponent<ProjectListItemProps> = ({
   state,
   project,
 }) => {
+  const classes = useStyles();
```
Comment (dwmorrin) [inline src/components/ProjectListItem.tsx:29]: no effect from removing

**Pull request merged**